### DISCUSSION TITLE
Adding a Audio device to AV Linux /Focusrite 18i20 hardware / Adding git hub project to Control hardware

### DISCUSSION THREAD

**Discussion by Rickyrivits** [community/community#77379] [Programming Help]
Not sure how to find answers.   New to Linux and Github so I need help with linux commands in terminal. Where do I post for help? 
I tested with Ubuntu Studio with the alsa-scarlett=gui and it works like a charm but AV Linux can compile.
So I need to figure out how to work arround the problems in AV linux https://www.bandshed.net/
The git hub package I am working with is: [geoffreybennett / alsa-scarlett-gui ](https://github.com/geoffreybennett/alsa-scarlett-gui)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Rickyrivits  and welcome to GitHub! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by Rickyrivits** [comment]
thanks
